mbed_flasher/main.py

```diff
@@ -486,7 +487,13 @@ def mbedflash_main():
     """
 
     cli = FlasherCLI()
-    exit(cli.execute())
+    retcode = cli.execute()
+    if retcode:
+        if retcode in failure_return_code_mapping_table:
+            cli.logger.error("Failed: %s", retcode)
```

Comment:
This will now print the int return code, that is probably not the intention?

maybe:
```
if retcode:
  try:
    cli.logger.error("Failed: %s", failure_return_code_mapping_table[retcode])
  except KeyError:
    cli.logger.error("Failed with unknown reason: %s", str(retcode))
```

Reply:
ah true - will change